src/components/ui/card/Card.vue

```diff
@@ -72,9 +87,11 @@ watchEffect(() => (showBody.value = !props.dropdownClosed));
           </span>
         </div>
         <div class="ui-card-header-content-button">
-          <Link v-if="actions" v-for="item in actions" :key="item.label" @click="item.onAction">
-            {{ item.label }}
-          </Link>
+          <div v-if="actions">
+            <Link v-for="item in actions" :key="item.label" @click="item.onAction">
+              {{ item.label }}
+            </Link>
+          </div>
```

Comment:
A adição dessa div pode gerar problemas de CSS, pra evitar qualquer efeito não esperado trocar para elemento não renderizável:
```suggestion
          <template v-if="actions">
            <Link v-for="item in actions" :key="item.label" @click="item.onAction">
              {{ item.label }}
            </Link>
          </template>
```